src/main/resources/db/migration/V18_12__add_column_is_aktiv_kartleggingssporsmal_vurdering.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE person_oversikt_status
+    ADD COLUMN is_aktiv_kartleggingssporsmal_vurdering BOOLEAN DEFAULT FALSE;
```

Comment:
Funderer litt på om det burde være NOT NULL her, men ser at det er gjort på den samme måten tidligere. Så kanskje best å la det være sånn.

Reply:
Enig i at det burde være `NOT NULL` ja. De andre skal jo egentlig ikke kunne settes til `null`, men nå ser det jo ut som vi egentlig har den muligheten 🫡